packages/mercury/src/components/chat/_chat-styles.scss

```diff
@@ -1,501 +1,867 @@
+    }
+    &--active {
+      --icon-path: var(--icon__system_link_neutral--active);
+    }
+    &--before {
+      // WA/TODO: Ask design to include a vertical attach icon on Figma.
+      @include icon-size--md();
+      transform: rotate(90deg);
+    }
   }
-}
 
-// send
-%send-container {
-  background-color: var(--mer-surface__elevation--02);
-  border-radius: 4px;
-  gap: $sendContainerGap;
-  margin: $commonPadding;
-  padding: 10px;
-}
-%send-input {
-  --ch-edit-gap: var(--control__icon-gap);
-  --ch-edit__image-size: var(--mer-icon__box--md);
-  --ch-placeholder-color: var(--control-placeholder__color);
-  --control__border-width: 0 !important;
-  --control__padding-block: 7px !important;
-  background-color: transparent !important;
-  border-color: var(--mer-color__neutral-gray--900);
-  border-radius: var(--control__border-radius);
-  border-style: var(--control__border-style);
-  border-width: var(--control__border-width);
-  color: var(--control__color);
-  padding-block: var(--control__padding-block);
-  padding-inline: var(--control__padding-inline);
-
-  &--hover {
-    --control__border-color: var(--control__border-color--hover);
-  }
-  &--focus {
-    outline: var(--focus__outline-width) var(--focus__outline-style)
-      var(--focus__outline-color);
-    outline-offset: var(--focus__outline-offset);
+  &__attached-files {
+    &__viewer {
+      grid-area: file-viewer;
+      @include reset-list();
+      display: grid;
+      grid-template-columns: repeat(3, 1fr);
+      grid-auto-rows: auto;
+      column-gap: var(--mer-spacing--2xs);
+      row-gap: var(--mer-spacing--2xs);
+      margin-inline-start: var(--mer-spacing--sm);
+      padding-block-start: var(--mer-spacing--xs);
+
+      > li {
+        border: var(--mer-border__width--sm) solid
+          var(--mer-border-color__on-elevation--02);
+        padding: var(--mer-spacing--xs);
+        border-radius: var(--mer-border__radius--sm);
+        background-color: var(--mer-surface__elevation--03);
+        @extend %body-regular-xs;
+        text-align: start;
+        @include ellipsis();
+      }
+    }
   }
-}
-%send-button,
-%stop-generating-answer-button {
-  &--before {
-    content: "";
-    display: inline-block;
-    inline-size: var(--mer-icon__box--md);
-    block-size: var(--mer-icon__box--md);
-    background: no-repeat center / var(--mer-icon__size--md) var(--icon-path);
+
+  &__files-container {
+    display: grid;
+    grid-auto-rows: max-content;
+    gap: var(--mer-spacing--xs);
+    margin-block-start: var(--mer-spacing--xs);
+  }
+
+  // send
+
+  &__send-container {
+    background-color: var(--mer-surface__elevation--02);
+    border-radius: var(--mer-spacing--2xs);
+    position: relative;
+    // gap: $sendContainerGap;
+    margin: $commonPadding;
+    // padding: 10px; // No token available for 10px at the time
+  }
+
+  &__send-input-wrapper {
+    position: relative;
+  }
+
+  &__send-input {
```

Comment:
The send-input doesn't have a max-block-size that should be customized with a custom var that determines the max-lines (this should be documented in the showcase).

```suggestion
  &__send-input {
    max-block-size: calc(padding-and-border-values + line-height-value * var(--mer-chat-send-input-max-lines, 12));
```

<img width="496" height="660" alt="image" src="https://github.com/user-attachments/assets/e67daaed-3d1c-405d-9d39-db71303366ce" />

Reply:
This should also be a screenshot test.